components/filter/filter.css

```diff
@@ -12,9 +12,11 @@
 .ecom-filter{
     width: 15rem;
     height:100vh;
-    background-color:rgb(212, 205, 205);
     padding:1rem;
     box-shadow: rgba(0, 0, 0, 0.16) 0px 10px 36px 0px, rgba(0, 0, 0, 0.06) 0px 0px 0px 1px;
```

Comment:
Just a suggestion, you can create different variables for box-shadows and then use them in your code. Otherwise, it looks good

Reply:
alright ,thanks.
I will change it